config/ember-try.js

```diff
@@ -18,14 +18,22 @@ module.exports = async function() {
         },
         npm: {
           devDependencies: {
-            'ember-source': null
+            'ember-source': null,
+            'ember-factory-for-polyfill': '1.3.1'
           },
+          dependencies: {
+            'ember-compatibility-helpers': '1.2.1'
```

Comment:
Do we need this in ember-try since it's a dependency of the addon already now?

Reply:
Yes, this is required. I confirmed it by dropping it locally and running the try config.

The dependency in the `package.json` is `^1.2.1`, where this is `1.2.1`. Wait wait, I agree, the `resolutions` should take care of that, however practically it doesn't do the trick on its own.